### PR TITLE
DC-641 Update Custom VCC API docs

### PR DESCRIPTION
**Pull request by eltesoro-koji**

### CODE REVIEW

File: src/asciidoc/reference/customVcc/withkojiCustomVccSdk.adoc [hunk 1]
```diff
@@ -58,10 +58,10 @@ customVcc.save();
 
 === Using custom controls
 
-To make a custom control accessible to Koji templates, you must serve it at a custom domain.
+To make a custom control accessible to Koji app, you must serve it at a custom domain.
```
Comment (rasienko) [inline src/asciidoc/reference/customVcc/withkojiCustomVccSdk.adoc:61]:
```suggestion
To make a custom control accessible to other Koji apps, you must serve it at a custom domain.
```

File: src/asciidoc/reference/customVcc/customVccReference.adoc [hunk 2]
```diff
@@ -152,7 +152,7 @@ However, this object can be useful when creating more complex custom controls, l
```
Comment (rasienko) [inline src/asciidoc/reference/customVcc/customVccReference.adoc:152]: In Interfaces > Props > Properties, there's an extra underscore before the type of `config`, which should be `_config`. We probably need to escape the first underscore (and no one noticed it in previous drafts).

**Review by rasienko — APPROVED** [review]
Looks good! Two little typos (one was just an existing problem that I happened to notice today). After this is merged, we'll update the readme in the package repo too. Thanks!

File: src/asciidoc/reference/customVcc/withkojiCustomVccSdk.adoc (hunk 1 above)
Comment (eltesoro-koji) [reply src/asciidoc/reference/customVcc/withkojiCustomVccSdk.adoc:61]: Fixed.

File: src/asciidoc/reference/customVcc/customVccReference.adoc (hunk 2 above)
Comment (eltesoro-koji) [reply src/asciidoc/reference/customVcc/customVccReference.adoc:152]: Removed the '_' before `config`.

Comment (rasienko) [reply src/asciidoc/reference/customVcc/customVccReference.adoc:152]: In the package code, the property name is `_config`, so we might need that character. If so, we can escape it so it doesn't mess with the Asciidoc markup. E.g. 
```
`+_config+` – _Object_, 
```

Comment (eltesoro-koji) [reply src/asciidoc/reference/customVcc/customVccReference.adoc:152]: Putting back the '_'. I don't understand your original comment then about an extra underscore.

**Review by rasienko — APPROVED** [review]
LGTM!

**Comment by eltesoro-koji** [comment]
Thank you! Merged.

**Pull request merged**